geomesa-tools/bin/download-data.sh

```diff
@@ -53,14 +53,16 @@ case "$type" in
   tdrive)
 
     echo "Note: each zip file contains approximately one million points"
-    read -p "Download how many zip files? " NUM
+    read -p "Download how many zip files? (9 total) " NUM
```

Comment:
They may have grouped some them together, but i don't feel like their claim of 15 million points still holds.
[See here](https://www.microsoft.com/en-us/research/publication/t-drive-trajectory-data-sample/)